packages/components/atoms/f-button/src/components/_tests/Button.test.js

```diff
@@ -237,6 +280,45 @@ describe('Button', () => {
                     expect(wrapper.attributes('type')).toBeUndefined();
                 });
             });
+
+            // RouterLink type -> incl. `to` attribute
+            describe('when `to` prop is empty :: ', () => {
+                it('should add `type` attribute', () => {
```

Comment:
More a Q than a comment – is this `type="button"` that gets added?

Reply:
Yes, in this case of `to` prop empty, it renders the default `<button>` element with type attribute.

Reply:
Found an issue when I dug up deeper, an attribute of `action-type` was added. Now fixed. 
![Screenshot 2021-06-15 at 17 08 58](https://user-images.githubusercontent.com/26770126/122092654-14d9b300-ce02-11eb-8383-79138426bff5.png)

Reply:
Should this test be the same as one for `href` checking for both `type` and `action-type`? I think you can use [describe.each()](https://jestjs.io/docs/api#describeeachtablename-fn-timeout) in this case to remove duplication